packages/client/src/service/request/request.ts

```diff
@@ -0,0 +1,93 @@
+enum METHODS {
+  GET = 'GET',
+  POST = 'POST',
+  PUT = 'PUT',
+  DELETE = 'DELETE'
+}
+
+interface Options {
+  method?: string
+  headers?: Record<string, string>
+  data?: unknown
+  timeout?: number
+}
+
+type HTTPMethod = (url: string, options: Options, timeout?: number) => Promise<unknown>;
+
+const TIMEOUT_DELAY = 5000;
+const HOST = 'https://ya-praktikum.tech/api/v2';
+
+function queryStringify(data: Record<string, string>): string {
+  if (typeof data !== 'object') {
+    throw new Error('Data должна быть объектом');
+  }
```

Comment:
[Предложение]
Можно отдавать data = undefined и возвращать в таком случае пустую строку, чтобы не использовать тернарный оператор ниже
```suggestion
function queryStringify(data?: Record<string, string>): string {
  if (!data) return '';
  if (typeof data !== 'object') {
    throw new Error('Data должна быть объектом');
  }
```